iron-resizable-behavior.html

```diff
@@ -61,6 +61,20 @@
     },
 
     attached: function() {
+      // NOTE(valdrin) In CustomElements v1 with native HTMLImports, the order
+      // of imports affects the order of `attached` callbacks (see webcomponents/custom-elements#15).
+      // This might cause a child to notify parents too early (as the parent
+      // still has to be upgraded), resulting in a parent not able to keep track
+      // of the `_interestedResizables`. To solve this, we wait for the document
+      // to be done loading before firing the event.
+      if (document.readyState === 'loading') {
+        var _this = this;
+        document.addEventListener('readystatechange', function readystatechanged() {
+          document.removeEventListener('readystatechange', readystatechanged);
+          _this.isAttached && _this.attached();
+        });
+        return;
+      }
```

Comment:
(from offline disussion) This particular example above is currently a problem in the 1.x version of the behavior and would require significant changes to fix so I don't think we should attempt to fix it here. We should consider fixing it in the future; I think I know a reasonably efficient way to do this too.